src/collection.js

```diff
@@ -71,6 +71,9 @@ export default class Collection {
     this.db = db;
     this.api = api;
     this.events = options.events || new EventEmitter();
+    this.plugins = {
```

Comment:
Here, I used `this.remoteTransformers` instead of `this.options.remoteTransformers`, for consistency with for instance the line just above, which also uses `this.events` and not `this.options.events`.